### PR TITLE
fix(Field.Toggle): removes invisible label when label not provided

### PR DESCRIPTION
**Pull request by langz**
Fixes https://jira.tech.dnb.no/browse/DCE-12453

Before this PR:
image

After this PR:
https://github.com/dnbexperience/eufemia/pull/3984/files#diff-d4c280bed02d42be9b1a172ed18eea80d2cb23e0a33ad9dd27b88847815d5177

image

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/base-fields/Toggle/demos.mdx [hunk 1]
```diff
@@ -66,6 +66,12 @@ import * as Examples from './Examples'
 
 <Examples.VariantButtonsWithHelp />
 
+#### Buttons without label
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/base-fields/Toggle/demos.mdx:69]:
```suggestion
```

**Review by joakbjerk — APPROVED** [review]

**Review by tujoworker — APPROVED** [review]
I would rather move the logic into the `FieldBlock` itself. Because other fields may suffer from the same. I pushed a commit with tests. 

I could not find any `isFragment` or `fragmentHasChildren`, so added for now to the FieldBlock.

**Comment by langz** [comment]
> I would rather move the logic into the `FieldBlock` itself. Because other fields may suffer from the same. I pushed a commit with tests.
> 
> I could not find any `isFragment` or `fragmentHasChildren`, so added for now to the FieldBlock.

Great, thanks.
I see that the existing fix does not work anymore now, I'll look into it.
Now it looks like this(as it did before the fix):
image

**Comment by langz** [comment]
> > I would rather move the logic into the `FieldBlock` itself. Because other fields may suffer from the same. I pushed a commit with tests.
> > I could not find any `isFragment` or `fragmentHasChildren`, so added for now to the FieldBlock.
> 
> Great, thanks. I see that the existing fix does not work anymore now, I'll look into it. Now it looks like this(as it did before the fix): image

I've added a fix for this now, in the following commit https://github.com/dnbexperience/eufemia/pull/3984/commits/7fa80f515409f51cc562afb17835867c6d66f9bc

Happy with how the fix looks now 🙏

**Comment by langz** [comment]
I also had to update screenshot tests in this commit  https://github.com/dnbexperience/eufemia/pull/3984/commits/68370f6b4f2edc2c3abd7980ea33aea63b47140e

Looks good to me:
image

**Pull request merged**